store/postgres/src/connection_pool.rs

```diff
@@ -1024,6 +1012,20 @@ impl PoolInner {
         });
         result.unwrap_or_else(|err| die(&pool.logger, "migrations failed", &err));
         debug!(&pool.logger, "Setup finished"; "setup_time_s" => start.elapsed().as_secs());
+
+        if let Err(msg) = catalog::Locale::load(&conn)?.suitable() {
+            if &self.shard == &*PRIMARY_SHARD && primary::is_empty(&conn)? {
+                die(
+                    &pool.logger,
+                    "Database does not use C locale. \
+                    Please check the graph-node documentation for how to set up the database locale",
+                    &msg,
+                );
+            } else {
+                warn!(pool.logger, "{}.\nPlease check the graph-node documentation for how to set up the database locale", msg);
+            }
+        }
+
```

Comment:
To help with the clarity of logging, it would be better to have this check just before "Setup finished" is logged.

Reply:
Makes sense.
Fixed in 3ca714f83f221519931f208eab124bc1cf56adde